### PR TITLE
Docs for v3.3

### PR DESCRIPTION
**Pull request by adrianeboyd**
## Description

Docs for v3.3

### Types of change

Docs.

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [ ] I ran the tests, and all new and existing tests passed.
- [ ] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: website/docs/models/index.md [hunk 1]
```diff
@@ -154,10 +187,31 @@ nlp.enable_pipe("senter")
 The `senter` component is ~10&times; faster than the parser and more accurate
 than the rule-based `sentencizer`.
 
+#### Switch from trainable lemmatizer to default lemmatizer
+
+A number of v3.3 pipelines use a new trainable lemmatizer. You can check whether
```
Comment (svlandeg) [inline website/docs/models/index.md:192]: I would rephrase slightly to make this more future-proof:
```suggestion
Since v3.3, a number of pipelines use a trainable lemmatizer. You can check whether
```

File: website/docs/models/index.md [hunk 2]
```diff
@@ -154,10 +187,31 @@ nlp.enable_pipe("senter")
...
+
+If you'd like to switch to a non-trainable lemmatizer that's similar to v3.2 or
+earlier, you can replace the trainable lemmatizer with the default non-trainable
+lemmatizer:
+
+```python
+# Requirements: pip install spacy-lookups-data
+nlp = spacy.load("de_core_web_sm")
+nlp.remove_pipe("lemmatizer")
+nlp.add_pipe("lemmatizer").initialize()
```
Comment (svlandeg) [inline website/docs/models/index.md:208]: Hmmm the naming is still a bit confusing here, I think. Should we clarify this in the docs that they are from different factories and that the trainable one has been renamed for convenience?

Maybe it helps if we also show how to switch from a non-trainable one to a trainable one?

File: website/docs/usage/v3-3.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+nlp = spacy.blank("en")
+text = "Welcome to the Bank of China."
+doc = nlp(text)
+doc.spans["custom"] = [Span(doc, 3, 6, "ORG"), Span(doc, 5, 6, "GPE")]
+displacy.serve(doc, style="span", options={"spans_key": "custom"})
+```
+
+import DisplacySpanHtml from 'images/displacy-span.html'
+
+<Iframe title="displaCy visualizer for entities" html={DisplacySpanHtml} height={180} />
```
Comment (svlandeg) [inline website/docs/usage/v3-3.md:90]:
```suggestion
<Iframe title="displaCy visualizer for overlapping spans" html={DisplacySpanHtml} height={180} />
```

File: website/docs/usage/v3-3.md [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+| [`ko_core_news_md`](/models/ko#ko_core_news_md) |              - |           90.0 |
+| [`lt_core_news_md`](/models/lt#lt_core_news_md) |           71.1 |           84.8 |
+| [`nb_core_news_md`](/models/nb#nb_core_news_md) |           76.7 |           97.1 |
+| [`nl_core_news_md`](/models/nl#nl_core_news_md) |           81.5 |           94.0 |
+| [`pl_core_news_md`](/models/pl#pl_core_news_md) |           87.1 |           93.7 |
+| [`pt_core_news_md`](/models/pt#pt_core_news_md) |           76.7 |           96.9 |
+| [`ro_core_news_md`](/models/ro#ro_core_news_md) |           81.8 |           95.5 |
+| [`sv_core_news_md`](/models/sv#sv_core_news_md) |              - |           95.5 |
+
+</figure>
```
Comment (svlandeg) [inline website/docs/usage/v3-3.md:143]: Pretty impressive table :-)

File: website/docs/models/index.md (hunk 2 above)
Comment (adrianeboyd) [reply website/docs/models/index.md:208]: Maybe just clarify that this adds the non-trainable lemmatizer from the language defaults.

I don't want to try how to switch to a trainable lemmatizer in an existing pipeline because there are too details (frozen, which tok2vec, etc.).

File: website/docs/usage/v3-3.md (hunk 3 above)
Comment (adrianeboyd) [reply website/docs/usage/v3-3.md:82]: Already done internally when working on the blog post, but I'll fix it in the linked main docs, too.
